### ISSUE TITLE
Failing CI

### ISSUE THREAD

**Issue by DanielRuf** [verdaccio/ui#46]
**Describe the bug**
It seems this is some sort of race condition. At least I can not reproduce it locally.
https://github.com/verdaccio/ui/runs/117059984

```
error /github/workspace/node_modules/node-sass: Command failed.
Exit code: 1
Command: node scripts/install.js
Arguments: 
Directory: /github/workspace/node_modules/node-sass
Output:
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module './lib/har'
    at Object.<anonymous> (/github/workspace/node_modules/request/request.js:23:11)
```

**To Reproduce**
1. Check the current CI status

**Expected behavior**
CI status should be green.

**Comment by DanielRuf** [comment]
Also weird to see

```
warning " > @verdaccio/types@5.0.0-beta.4" has unmet peer dependency "@types/node@10.12.18".
warning " > @verdaccio/types@5.0.0-beta.4" has unmet peer dependency "typescript@3.2.4".
warning " > @verdaccio/types@5.0.0-beta.4" has incorrect peer dependency "flow-bin@0.90.0".
warning " > jest-environment-jsdom-global@1.1.1" has incorrect peer dependency "jest-environment-jsdom@22.x || 23.x".
warning " > stylelint-webpack-plugin@0.10.5" has incorrect peer dependency "stylelint@^8.0.0".
```

Not sure if we have to upgrade Yarn.

**Comment by juanpicado** [comment]
Those dependencies, that's my fault :)

**Comment by juanpicado** [comment]
First time I see this tbh.
```
[5/5] Building fresh packages...
error /github/workspace/node_modules/node-sass: Command failed.
Exit code: 1
Command: node scripts/install.js
Arguments: 
Directory: /github/workspace/node_modules/node-sass
Output:
internal/modules/cjs/l
````

**Comment by juanpicado** [comment]
I think GitHub actions are a bit buggy ... @ayusharma has noticed some mysterious bugs lately.

**Comment by DanielRuf** [comment]
Not sure what you mean as the last commit did introduce any dependency changes.

**Comment by juanpicado** [comment]
Ok, some context.

* I introduced `babel-preset` to simplify dependencies overall projects. In such dependency, pull babel and other dependencies required to build the app, some of them are optional dependencies which are generating those warnings, that part is my fault.
* We removed CircleCI from this project, trying to introduce GitHub Actions, but we were experimenting. We are thinking to restore such config and give more time to GitHub actions which will run as non required steps (cc: @ayusharma )

Does it make sense now? :-)
